src/structures/Player.ts

```diff
@@ -126,6 +126,13 @@ export class Player {
     this.manager.players.set(options.guild, this);
     this.manager.emit("playerCreate", this);
     this.setVolume(options.volume ?? 100);
+    this.manager.on('nodeDisconnect', (node) => {
```

Comment:
Possibly have an if statement wrapping this? If `replayOnDc` is false you'll just be adding a redundant listener